rest-api/common-rest/src/main/java/uk/gov/gchq/gaffer/rest/model/OperationDetail.java

```diff
@@ -87,11 +88,15 @@ private static List<OperationField> getOperationFields(final Class<? extends Ope
     }
 
     private static String getOperationOutputType(final Operation operation) {
-        String outputClass = null;
-        if (operation instanceof Output) {
-            outputClass = JsonSerialisationUtil.getTypeString(((Output) operation).getOutputTypeReference().getType());
+        try {
+            String outputClass = null;
```

Comment:
The changes here are not needed when https://github.com/gchq/Gaffer/pull/2756 is merged